jastrow.py

```diff
@@ -25,12 +49,36 @@ def eidist(configs, coords):
     return d
     
 
+def eidist(configs, coords, nup, ndown):
+    """returns a list of electron-ion distances"""
```

Comment:
Update the documentation.

Reply:
Documentation now mentions spin cases and how nup, ndown are used.